app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Acebook</title>
-    <%= csrf_meta_tags %>
+<head>
+  <title>Acebook</title>
+  <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
-    <%= javascript_include_tag 'application' %>
-  </head>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
 
-  <body>
-    <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
-      <%= button_to 'Sign out', sign_out_path, method: :delete %>
-    <% else %>
-      <%= link_to 'Sign in', sign_in_path %>
-    <% end %>
+<body>
+  <nav>
+    <a href="/">
+      <div id="page-logo">
+      </div></a>
+      <% if signed_in? %>
+      <div class="dropdown">
+        <li class="dropbtn"><i class="fa fa-chevron-circle-down" style="font-size:20px"></i></li>
+        <div class="dropdown-content">
+          <a href="/">Profile</a>
+          <a href="/">Account</a>
+          <%= link_to 'Sign out', sign_out_path, method: :delete %></a>
+        </div>
+      </div>
+      <li><a href="/" class="right-link" >Home</a></li>
+      <li><span class="right-link"> Signed in as: <%= current_user.email %></span></li>
 
-    <div id="flash">
-      <% flash.each do |key, value| %>
-        <div class="flash <%= key %>"><%= value %></div>
+      <% else %>
+      <li>  <%= link_to 'Sign in', sign_in_path, { :class=>"right-link" } %></li>
```

Comment:
So there are a few rules around `li` elements you should be aware of.

* `<li>` elements must be directly inside a `<ul>` or an `<ol>` element.
* `<ul>` and `<ol>` elements *direct children* can only be `<li>` elements.

So these are valid:

```html
<ul>
  <li>Hello!</li>
  <li>Hi!</li>
</ul>

<ul>
  <li><div>Woo!</div></li>
</ul>
```

But these are not:

```html
<div>
  <li>Noo!</li>
</div>

<ul>
  <div>
    <li>Hey no!</li>
    <li>Oops!</li>
  </div>
</ul>
```

The logic being — `ul` is an Unordered List and lists can only contain `li` (List Items). Other arrangements 'don't make sense' according to the spec.

Take a look at this stackoverflow thread for some discussion on this: https://stackoverflow.com/questions/4230983/can-you-have-lis-without-them-being-under-a-ul-or-ol . You'll probably need to amend some of the CSS to make it still look alright after the changes!